azure-pipelines.yml

```diff
@@ -21,6 +22,8 @@ jobs:
   - script: |
       sudo apt --assume-yes install libusb-1.0-0-dev
       python3 -m pip install -r ./inference-engine/ie_bridges/python/requirements.txt
+      # For Python API tests, should be in requirements.txt
+      python3 -m pip install pytest==5.4.2
```

Comment:
there are should be dedicated requirements.txt file in tests folder

Reply:
agree, need a request on developers

Reply:
We have special requirements-dev.txt: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/ie_bridges/python/src/requirements-dev.txt